azure-pipelines.yml

```diff
@@ -160,7 +169,7 @@ jobs:
     timeoutInMinutes: 360
     pool:
       name: 'windows-pool'
-      demands: assignment -equals default
```

Comment:
I assume this is just temporary?

Reply:
Yes. My thinking is basically I'm mucking around with both Bazel and scoop so I may end up breaking the machine.

Reply:
Removed.